# https://gitweb.gentoo.org/repo/gentoo.git/commit/?id=6228d70137eab7594520fe6e4dbd1a9a23bfe967

.../lib/gcc/*-*-linux-gnu/1*.*.*/../../../../*-*-linux-gnu/bin/ld.bfd:
 tcdemux-demuxer.o:(.bss+0x0): multiple definition of `gop_cnt'; tcdemux-demux_pass.o:(.bss+0x0): first defined here
.../lib/gcc/*-*-linux-gnu/1*.*.*/../../../../*-*-linux-gnu/bin/ld.bfd:
 tcdemux-demuxer.o:(.bss+0x4): multiple definition of `gop_pts'; tcdemux-demux_pass.o:(.bss+0x4): first defined here
.../lib/gcc/*-*-linux-gnu/1*.*.*/../../../../*-*-linux-gnu/bin/ld.bfd:
 tcdemux-demuxer.o:(.bss+0x8): multiple definition of `gop'; tcdemux-demux_pass.o:(.bss+0x8): first defined here
collect2: error: ld returned 1 exit status

---

--- a/import/demux_pass.c
+++ b/import/demux_pass.c
@@ -32,8 +32,6 @@
 #include "packets.h"
 
 
-int gop, gop_pts, gop_cnt;
-
 /* if you listen carefully, then you can hear the desesperate
  * whisper of this code calling for a rewrite. Or for a redesign.
  * Or both. --  FR
--- a/import/extract_ac3.c
+++ b/import/extract_ac3.c
@@ -334,7 +334,6 @@ static void pes_ac3_loop (void)
 FILE *fd;
 
 #define MAX_BUF 4096
-char audio[MAX_BUF];
 
 
 /* from ac3scan.c */
@@ -472,6 +471,8 @@ static int ac3scan(int infd, int outfd)
 void extract_ac3(info_t *ipipe)
 {
 
+    char audio[MAX_BUF];
+
     int error=0;
 
     avi_t *avifile;
--- a/import/extract_mp3.c
+++ b/import/extract_mp3.c
@@ -243,7 +243,6 @@ static int mp3scan(int infd, int outfd)
 }
 
 #define MAX_BUF 4096
-char audio[MAX_BUF];
 
 /* ------------------------------------------------------------
  *
@@ -258,6 +257,7 @@ char audio[MAX_BUF];
 
 void extract_mp3(info_t *ipipe)
 {
+    char audio[MAX_BUF];
 
     int error=0;
 
